libraries/AP_Airspeed/examples/Airspeed/Airspeed.cpp

```diff
@@ -65,6 +65,7 @@ void setup()
     board_config.init();
 
     // initialize airspeed
+    airspeed.set_log_bit(false);
```

Comment:
`false` evaluates to zero; see same comment on AP_Periph.

If you don't want to log - just omit this call...

Reply:
I've added a comment instead of blanket removal since it is an example of usage